### PR TITLE
initialization of the project with components

### PR DESCRIPTION
**Pull request by GhostEsso**
- I made sure that there are no linter errors;
- I made sure that I used correct flow Git-flow;
- I made sure that I documented my work in a professional way;
-  I setup a new React application using Create React App;
-  I created a directory for my reusable components;
- I installed React Router V6;
- I setup my router inside of App;
- I added 2 routes in my application;
- I created components inside of the components directory;

### CODE REVIEW

File: src/components/BookList.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const BookList = () => (
+  <section className="section1">
+    <h2>Books here</h2>
+  </section>
+);
```
Comment (AmaduKamara) [inline src/components/BookList.js]: - [ ] Good job creating this component, however, in this component, you're expected to render individual book components and pass down props to it. Therefore, kindly create another component named Book.js and pass the title and author props to it and receive them inside it and render the title and author alongside a delete button inside that individual component. 👍🏽

File: src/App.js [hunk 2]
```diff
@@ -1,22 +1,18 @@
...
-          Learn React
-        </a>
-      </header>
+    <div className="main">
+      <NavigationBar />
+      <Routes>
+        <Route path="/" element={<BookList />} />
+        <Route path="/Categories" element={<Categories />} />
+      </Routes>
+      <AddBook />
```
Comment (AmaduKamara) [inline src/App.js:15]: - [ ] Kindly render the Addbook form inside the BookList component. At the moment, the form is rendered everywhere in the app even if I'm on the Categories page. Please move it to the BookList component and below the individual books. 👍🏽

```jsx
const BookList = () => (
  <section className="section1">
    <h2>Books here</h2>
    <div>
      <Book title="Book Title" author="Book Author" />
      <Book title="Book Title" author="Book Author" />
      <Book title="Book Title" author="Book Author" />
    </div>
    <AddBook />
  </section>
);
```

The code inside the BookList component should look like the code above 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @GhostEsso,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Used CRA to initialize the app 👍🏽
✅ The app runs successfully 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly create a Single Book component that is called inside the BookList component 👍🏽
- [ ] Pass props to the single Book component inside the BookList 👍🏽
- [ ] Render the props inside the single Book component alongside a button to delete a book. The delete button will do nothing at this moment, but later. 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/BookList.js [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
+import Book from './Book';
+import './BookList.css';
+
 const BookList = () => (
-  <section className="section1">
-    <h2>Books here</h2>
-  </section>
+  <>
+    <Book title="Book title" author="Book author" />
+    <button type="button" id="delete-button">Remove</button>
```
Comment (TadesseDev) [inline src/components/BookList.js:9]: - [ ] Dear @GhostEsso, Good job addressing most of the changes requested by the previous reviewer. But as suggested by the reviewer here, we need to make use of the `book.js` component and render individual books using the `book.js` component by passing the book/book detail as a prop.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
X

**Review by TadesseDev — APPROVED** [review]
### Hi @GhostEsso 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅


## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**